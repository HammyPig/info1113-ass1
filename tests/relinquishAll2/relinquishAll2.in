Relinquish All 100000
Add Member its a me, mario
Relinquish All 100000
Relinquish All 100001
Add Collection books.csv
Rent 100000 111111
Member Books 100000
Relinquish All 100000
Member Books 100000
Rent 100000 111111
Rent 100000 111112
Rent 100000 111113
Rent 100000 111113
Member Books 100000
Relinquish All 100000
Member Books 100000
Add member mrstealurbooks
Rent 100001 111111
Rent 100001 111112
Rent 100001 111113