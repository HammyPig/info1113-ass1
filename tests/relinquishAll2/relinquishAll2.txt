recognises if members/books do not exist, or if books are not being rented, otherwise relinquishes all as intended