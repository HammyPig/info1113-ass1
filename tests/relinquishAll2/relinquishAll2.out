user: No members in system.

user: Success.

user: Success.

user: No such member in system.

user: 17 books successfully added.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: Member not currently renting.

user: Success.

user: Success.

user: Success.

user: Book is currently unavailable.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: Success.

user: Member not currently renting.

user: Success.

user: Success.

user: Success.

user: Success.

user: Ending Library process.
